cylc/flow/parsec/config.py

```diff
@@ -53,8 +53,11 @@ def checkspec(spec_root, parents=None):
                     stack.append([value, pars])
                 elif not isinstance(value, list):
                     raise ParsecError(
-                        "Illegal file spec item: %s" % itemstr(
-                            pars, repr(value)))
+                        f"Illegal file spec item: "
+                        + itemstr(pars, repr(value))
+                        + " "
+                        + repr(value)
+                    )
```

Comment:
I think here we can either use normal strings, as the f-string is not actually used to replace any value?

Or maybe something like (not tested)

```
                    raise ParsecError(
                        f"""Illegal file spec item: 
                        {itemstr(pars, repr(value))} {repr(value)}"""
                    )
```

?